test/langtools/jdk/javadoc/doclet/testSnippetTag/TestSnippetTag.java

```diff
@@ -100,7 +95,7 @@ public static void main(String... args) throws Exception {
      * https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-code-element
      */
     @Test
-    public void testPositiveInlineTagIdAndLangAttributes(Path base) throws IOException {
+    public void testPositiveInlineTag_IdAndLangAttributes(Path base) throws IOException {
```

Comment:
General comment, applying to all such situations ... I like the latest iteration on the naming scheme, with the embedded `_`.